test_pull_requests

```diff
@@ -948,15 +1013,39 @@ popd
                   break
                 end
               end
+              # the main pull request and all of the co-requisite pull requests haven't been
+              # updated since the last trusted trigger, so we can begin testing
               update_evaluated_markers(repo_to_pull_request, trigger_updated_at, settings)
               submit_test_job = true
             else
+              # no trusted trigger statement has been made, so we cannot build or test this pull
+              # TODO: only pull requests with trusted triggers get process_pull_request called
```

Comment:
I am not sure this is true.  ppr is called when you know you have a comment from a trusted user.  But I don't think we know if that comment is at a place we can consider it a trusted trigger yet.